src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,163 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private static RegistrationService registrationService;
+    private User validUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        validUser = new User();
+        validUser.setId(1L);
+        validUser.setAge(22);
+        validUser.setLogin("validLogin");
+        validUser.setPassword("password");
+    }
+
+    @Test
+    void register_validUser_Ok() {
+        User registeredUser = registrationService.register(validUser);
+        assertEquals(validUser, registeredUser);
+        assertEquals(validUser.getLogin(), registeredUser.getLogin());
+        assertEquals(validUser.getAge(), registeredUser.getAge());
+        assertEquals(validUser.getId(), registeredUser.getId());
+        assertEquals(validUser.getPassword(), registeredUser.getPassword());

```

Comment:
This is redundant because we already check for this in the overridden equals method.